src/Relude/Numeric.hs

```diff
@@ -0,0 +1,76 @@
+{-# LANGUAGE RankNTypes          #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications    #-}
+
+{- |
+Copyright: (c) 2018 Kowainik
+License:    MIT
+Maintainer: Kowainik <xrom.xkov@gmail.com>
+-}
+
+module Relude.Numeric
+       ( module Data.Bits
+       , module Data.Int
+       , module Data.Word
+       , module Numeric
+       , module GHC.Float
+       , module GHC.Num
+       , module GHC.Real
+       , integerToBounded
+       , integerToNatural
+       )
+       where
+
+import Numeric
+
+import Data.Bits (toIntegralSized, xor)
+import Data.Function (($))
```

Comment:
Let's put this import under `Relude` section and import as `Relude.Function (($))` to not mix reexports and internal imports.